### DISCUSSION TITLE
[Rule Suggestion] avoid_var

### DISCUSSION THREAD

**Discussion by TesteurManiak** [Floating-Dartists/fd_lints#8] [Ideas]
Prefer to set explicitly the type when declaring a variable instead of `var` (especially to avoid type inference error):

**BAD**
```dart
var myVar = false;
```

**GOOD**
```dart
bool myVar = false;
```

**Comment by Pierre-Monier** [comment]
Yes this is a good one, and it is use by the default flutter analysis configuration if I remember correctly
